### PR TITLE
backup: add getLastest test for filebackend

### PR DESCRIPTION
**Pull request by xiang90**
/cc @hongchaodeng

### CODE REVIEW

File: pkg/backup/file_backend_test.go [hunk 1]
```diff
@@ -22,6 +22,49 @@ import (
 	"testing"
 )
 
+func TestFileBackendGetLatest(t *testing.T) {
+	names := []string{
+		makeBackupName("3.0.1", 12),
```
Comment (xiang90) [inline pkg/backup/file_backend_test.go]: we cannot. we need to ensure:

1. we ignore bad name
2. we ignore bad suffix
3. we ignore version
4. we get the file with large rev

Comment (hongchaodeng) [inline pkg/backup/file_backend_test.go]: > we ignore version

For this point, can we change `makeBackupName("3.0.0", 1),` to `makeBackupName("3.1.0", 1),` and add a comment "ignore version"?

Comment (xiang90) [inline pkg/backup/file_backend_test.go]: sure.

File: pkg/backup/file_backend_test.go [hunk 2]
```diff
@@ -22,6 +22,49 @@ import (
...
+		"3.0.1_18_etcd.tmp", // bad suffix
+		makeBackupName("3.0.1", 3),
+		makeBackupName("3.0.3", 19),
+		makeBackupName("3.0.0", 1),
+		"3.0.1_badbackup_etcd.backup", //bad backup name
```
Comment (hongchaodeng) [inline pkg/backup/file_backend_test.go]: space between `//` and `bad`

**Comment by hongchaodeng** [comment]
lgtm

**Comment by hongchaodeng** [comment]
Jenkins isn't working now.
Let me merge it and manually test them.

**Pull request merged**